services/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/builders/terraform/TFExecutor.java

```diff
@@ -0,0 +1,139 @@
+package org.eclipse.osc.orchestrator.plugin.huaweicloud.builders.terraform;
```

Comment:
how about wrapping an executor class, there will be more excutors like TFExecutor、RFExecutor、CloudFormationExecutor、PackerExecutor、APIExecutor...

Reply:
IMHO, these executors can be implemented as different packages in the `builders` directory. Thus an executor maybe is so complicated to be carried by a class object.

Reply:
ok, agree to extract some common methods for these executors and implement like this in different pkg. just some suggestion.